src/components/atoms/authors/authors.tsx

```diff
@@ -2,25 +2,33 @@ import { useEffect, useState } from 'react';
 import styles from './authors.module.scss';
 import { Author } from '../../../types';
 
-const authorLimit = 10;
+const authorLimit = 3;
+const authorLimits = [authorLimit, 10];
 
 export const Authors = ({ authors }: { authors: Author[] }): JSX.Element => {
   const [expanded, setExpanded] = useState<boolean | null>(null);
 
   useEffect(() => setExpanded(false), []);
-  const displayAuthors = authors.slice(0, expanded !== false ? authors.length : authorLimit);
-  const expansionText = expanded ? 'show less' : `show ${authors.length - authorLimit} more`;
 
   return (
-    <div className={styles.authors}>
-      <ol className={styles['authors-list']} aria-label="Authors of this article">
-        { displayAuthors.map(({ givenNames, familyNames }, index) => (
+    <div className={
+      `
+      ${styles.authors}${authors.length > authorLimit && expanded !== null ? ` ${styles['authors--collapsible']}` : ''}
+      ${expanded !== null ? authorLimits.map((limit) => (limit < authors.length ? ` ${styles[`authors--limit-${limit}`]}` : '')).join(' ') : ''}
+      `
+    }>
+      <ol className={`${styles['authors-list']}${expanded ? ` ${styles['authors-list--expanded']}` : ''}`} aria-label="Authors of this article">
+        { authors.map(({ givenNames, familyNames }, index) => (
           <li className={styles['authors-list__item']} key={index}>
             {givenNames.join(' ')} {familyNames.join(' ')}
           </li>
         ))}
       </ol>
-      {(authors.length > authorLimit && expanded !== null) && <span className={styles['authors-list__expansion']} onClick={() => setExpanded(!expanded)}>{expansionText}</span>}
+        {(authors.length > authorLimit && expanded !== null) &&
+        <span className={styles['authors-list__expansion']} onClick={() => setExpanded(!expanded)}>
+          show{!expanded ? authorLimits.map(
+          (limit, index) => <span key={index} aria-hidden="true" className={`${styles['authors-list__expansion-count']} ${styles[`authors-list__expansion-count-${limit}`]}`}> {authors.length - limit}</span>,
```

Comment:
After a chat with @discodavey we have added `aria-hidden`. The inclusion of the counts for screen readers will be confusing but the text without the count will still make sense.